scripts/create-new-actor.sh

```diff
@@ -151,26 +320,67 @@ struct ${actor_name}::PhaseConfig {
 void ${actor_name}::run() {
     for (auto&& config : _loop) {
         for (const auto&& _ : config) {
-            auto inserts = _totalInserts.start();
-            // TODO: main logic
+            //
+            // Evaluate the DocumentGenerator template:
+            //
             auto document = config->documentExpr();
+
+            //
+            // The clock starts running only when you call ${q}.start()${q}. The return value
+            // from this lets you record how many bytes, documents, and actual
+            // iterations (usually 1) your Actor completes while the clock is running. You then
+            // must stop the clock by calling either ${q}.success()${q} if everything is okay
+            // or ${q}.fail()${q} if there was a problem. It's undefined behavior what will
+            // happen if you don't stop the metrics clock (e.g. if there is an uncaught
+            // exception). See the full documentation on ${q}OperationContext${q}.
+            //
+            // We don't care about how long the value-generator takes to run
+            // so we don't start the clock until after evaluating it.
+            //
+            auto inserts = _totalInserts.start();
+
+            //
+            // You have the full power of boost logging which is rendered to stdout
+            //
             BOOST_LOG_TRIVIAL(info) << " ${actor_name} Inserting "
                                     << bsoncxx::to_json(document.view());
-            config->collection.insert_one(document.view());
-            inserts.addDocuments(1);
-            inserts.addBytes(document.view().length());
-            inserts.success();
+
+            //
+            // If your Actor throws any uncaught exceptions, the whole Workload will
+            // attempt to end as quickly as possible. Every time this inner loop loops
+            // around it checks for any other Actors' exceptions and will stop iterating
+            // if it sees any. Such failed workloads are considered "programmer error"
+            // and will mark the workload task as a system-failure.
```

Comment:
"Evergreen" system failure? I don't know if "system-failure" by itself is a thing